### PR TITLE
Give examples of closure compiler annotations

### PR DESCRIPTION
**Pull request by ackh**
Issue #13827 could have been prevented if the documentation showed how to instruct the closure compiler to not rename JavaScript methods. This pull request is a direct result from that issue and amends the documentation accordingly

### CODE REVIEW

File: site/source/docs/porting/connecting_cpp_and_javascript/embind.rst [hunk 1]
```diff
@@ -173,6 +173,23 @@ shown below:
...
+
+In order to prevent the closure compiler from renaming the symbols in the
+above example code it needs to be rewritten as follows:
+
+   var instance = new Module["MyClass"](10, "hello");
+   instance["incrementX"]();
+   instance["x"]; // 11
+   instance["x"] = 20; // 20
+   Module["MyClass"]["getStringFromInstance"](instance); // "hello"
+   instance.delete();
```
Comment (kripken) [inline site/source/docs/porting/connecting_cpp_and_javascript/embind.rst:193]:
```suggestion
Note that this is only needed for code seen by the optimizer, for example as in
``--pre-js`` or ``--post-js`` as mentioned above, or on ``EM_ASM`` or ``EM_JS``.
For other code, that is not optimized by closure compiler, you do not need to
make such changes. You also do not need it if you build without ``--closure 1``
to enable the closure compiler.
```

**Review by kripken — COMMENTED** [review]
Thanks!

Please add yourself to AUTHORS. I also suggested another paragraph, what do you think?

**Comment by ackh** [comment]
Good addition from your side. I have added some empty lines to separate the paragraphs and added my name to the AUTHORS file.

**Review by kripken — APPROVED** [review]
Thanks!

**Pull request merged**